cmd/schemagen/schemagen/cli_all.go

```diff
@@ -27,8 +26,8 @@ func createAllSchemaFiles(components component.Factories, env env) {
 	}
 }
 
-func getAllConfigs(components component.Factories) []config.NamedEntity {
-	var cfgs []config.NamedEntity
+func getAllConfigs(components component.Factories) []interface{} {
```

Comment:
No more common interface?

Reply:
not yet, I will add one later. Since now I will change the configs one by one.

Reply:
what about `identifiable`? I see you didn't export though.

Reply:
This `cmd` actually does not need any common, but let me see how the config will look like and I will have some common interface.